.github/workflows/build.image.yml

```
@@ -0,0 +1,54 @@
+name: Build image
+
+on:
+    push:
+        branches:
+            - main
+        paths:
+            - 'requirements/*'
+    pull_request:
+        branches:
+            - main
+        paths:
+            - 'requirements/*'
+
+jobs:
+
+    set_vars:
+        uses: ./.github/workflows/set-vars.yml
+
+    build-image:
+        needs: [set_vars]
```

Let's call it `set-vars` :wink:
```suggestion
        needs: [set-vars]
```

---

More consistent indeed!